app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,57 @@
+class PasswordResetsController < ApplicationController
+  before_action :get_user, :valid_user, :check_expiration, only: %i(edit update)
```

Comment:
use 'load_user' or 'find_user' instead of 'get_user'
```suggestion
  before_action :load_user, only: %i(create edit update)
  before_action :valid_user, :check_expiration, only: %i(edit update)
...
def load_user
   email_param = request.post? ? params[:password_reset][:email] : params[:email]
    @user = User.find_by email: email_param
end
```